gateway/webrtc/client.js

```diff
@@ -213,19 +218,37 @@ loginBtn.addEventListener("click", function(event) {
     ownQueue = session.createTopic("/topic/" + name);
     console.log("Created queue : " + ownQueue);
     if (ownQueue !== '') {
-        startChat();
+        startChat(function() {
```

Comment:
It is possible to pass only the session as a parameter, but this way, all the JMS code is in a single place.